### PR TITLE
PXB-2749 Create 2.4.25 release notes and release note tasks

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file: storage/innobase/xtrabackup/doc/source/_res/replace/pxb-blurb.txt
new file: storage/innobase/xtrabackup/doc/source/_res/replace/useful-links.txt
modified: storage/innobase/xtrabackup/doc/source/conf.py
modified: storage/innobase/xtrabackup/doc/source/release-notes.rst
new file: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.25.rst

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]

**Review by george-lorch — APPROVED** [review]
+1

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.25.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Release Highlights
+=================================================
+
+The xbcloud binary adds support for the :ref:`Microsoft Azure Cloud Storage <xbcloud_azure>` using the REST API. 
+
+New Features
+=================================================
+
+* :jirabug:`PXB-1883`: Implements support for Microsoft Azure Cloud Storage in the xbcloud binary. (Thanks to Ivan for reporting this issue)
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.25.rst]:
```suggestion
* :jirabug:`PXB-1883`: Implements support for Microsoft Azure Cloud Storage in the xbcloud binary. (Thanks to Ivan Groenewold for reporting this issue)
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]
just a small change to put Ivan's full name on the thanks section.

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.25.rst (hunk 1 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.25.rst]: Fixed

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**